manifests/07_deployment.yaml

```diff
@@ -59,3 +59,6 @@ spec:
         operator: "Exists"
         effect: "NoExecute"
         tolerationSeconds: 120
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: "NoSchedule"
```

Comment:
I'm trying to understand if this operator must run exclusively on master nodes as well:

According to the [conventions doc](https://github.com/openshift/enhancements/blob/4cf172b4f405bf71b7789b14bbafc7e1855f099c/CONVENTIONS.md#taints-and-tolerations) operators managed by CVO should run on master nodes, so the tolerations must be properly set (which we're fixing in PR). It doesn't say anything about enforcing it (via nodeSelector for example), but in practice operators like KCM enforce that via nodeSelector:

https://github.com/bertinatto/cluster-kube-controller-manager-operator/blob/4ff72a19d344ce2bf09fa597e44d14a77a5a93ed/manifests/0000_25_kube-controller-manager-operator_06_deployment.yaml#L68-L69

So IMO we should specify nodeSelector here as well.

Reply:
yeah I was thinking of enforcing scheduling via selector for snapshot operator here. I will update the PR.

Reply:
The only problem I can think of is if the PodNodeSelector admission plugin is disabled, but then we'd have bigger problems...